File: back_end/petmily_pjt_temp/src/main/java/com/pjt/petmily/domain/curation/Curation.java
package com.pjt.petmily.domain.curation;


import com.pjt.petmily.domain.user.User;
import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import lombok.*;

import java.time.LocalDateTime;

@Builder
@Entity
@Getter
@Table(name="curation")
@NoArgsConstructor
@AllArgsConstructor
@Data
public class Curation {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long cId;

    @NotNull
    private String cTitle;

    @Column(nullable = true)
    private String cPetSpecies;

    @Column(nullable=true)
    private String cContent;

    @Column(nullable=true)
    private String cImage;

    @Column(nullable=true)
    private String cUrl;

    @Column(nullable = true)
    private String cDate;

    @Column
    private String cCategory;

    @Column
    private Integer cBookmarkCnt = 0;



}

File: mobile/Petmily/app/src/main/java/com/petmily/repository/dto/Curation.kt
package com.petmily.repository.dto

data class Curation(
    var cid: Long = 0L,
    var ctitle: String = "",
    var cpetSpecies: String = "",
    var ccontent: String = "",
    var cimage: String = "",
    var curl: String = "",
    var ccategory: String = "",
    var cdate: String = "",
)
